pkg/transport/udpmultimux.go

```diff
@@ -0,0 +1,73 @@
+package transport
```

Comment:
Is the idea to move all these into Pion when the API discussion there comes to a conclusion?

Reply:
Comment in the file that once the PR is merged, we can get rid of this implementation